include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -164,9 +166,39 @@ class BCX_API qrcode
             "The path to the configuration settings file."
         )
         (
+            "image,i",
+            value<bool>(&option_.image)->default_value(false)->zero_tokens(),
+            "Specifies that the QR code should be output in png format."
+        )
+        (
             "size,s",
             value<uint32_t>(&option_.size)->default_value(8),
-            "Specifies the module size in dots of the generated QR code."
+            "Specifies the module size in dots (pixles) of the generated QR code."
+        )
+        (
+            "margin,m",
+            value<uint32_t>(&option_.margin)->default_value(2),
+            "Specifies the margin size of the generated QR code."
+        )
+        (
+            "dpi,d",
+            value<uint32_t>(&option_.dpi)->default_value(72),
+            "Specifies the dpi of the generated QR code."
+        )
+        (
+            "version,v",
+            value<uint32_t>(&option_.version)->default_value(0),
+            "Specifies the version of the generated QR code."
+        )
+        (
+            "casesensitive,c",
+            value<uint32_t>(&option_.casesensitive)->default_value(1),
+            "Specifies the case sensitivity of the generated QR code."
```

Comment:
Is this is Boolean option or an enumeration? If the former we should revise the libbitcoin API to expose the option as a `bool` and if the latter we should define a `class enum` and expose it as such. If it's an enumeration we will want to create a primitive for it as well, for proper deserialization.